update 0.2.3

tlestang opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

tlestang committed:
update 0.2.3

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

tlestang committed:
Remove jax and notebook deps

tlestang committed:
Update sha256 to match sdist

tlestang committed:
update pybamm-feedstok to 0.2.3.post1
The CMakebuild.py was not included in the sdist for 0.2.3.
The sdist for 0.2.3.post1 includes it.

tlestang committed:
Update sha256

tlestang committed:
Update tests

tlestang committed:
Update tests for commands

tlestang commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 3.19.2, conda-smithy 3.7.4, and conda-forge-pinning 2020.07.27.09.21.49
